doc/classes/NavigationAgent2D.xml

```diff
@@ -229,17 +230,17 @@
 		</signal>
 		<signal name="navigation_finished">
 			<description>
-				Notifies when the final position is reached.
+				Emitted once per loaded path when the agent internal navigation path index reaches the last index of the loaded path array. The agent internal navigation path index can be received with [method get_current_navigation_path_index].
```

Comment:
`received with` -> `obtained using`

Reply:
Not sure, 'received with' is already used in the same class as well as other classes while 'obtained using' is not used anywhere except an entry in the RenderingServer doc.

Reply:
ok then